cluster-scope/README.md

```diff
@@ -1,27 +1,48 @@
 
-This is just a short addendum to [ADR 0009](https://www.operate-first.cloud/blueprints/blueprint/docs/adr/0009-cluster-resources.md) declaring the direct consequences and organization structure of this `cluster-scope` application. These are the best practises we expect every maintainer who does changes to this application to follow.
+In this directory we declaratively store cluster resources and other privileged resources that are not accessible by project admins.
 
 ## Application folder structure
 
-- `base`: Contains all the actual resource manifests which can be deployed to any clusters. Manifests are required to be stored in a format that can be applied to all the clusters (no cluster specific `.spec`s).
-  - `<RESOURCE_KIND_PLURAL_NAME>`: Each resource kind managed by this application has a folder using the resource name in plural format, e.g. `clusterroles` or `namespaces`.
-    - `<RESOURCE_NAME>`: The name of the resource deployed at cluster scope. It must match the resource `.metadata.name` in `<RESOURCE_KIND_SINGULAR_NAME>.yaml`. Since this resource is deployed on cluster scope, its name has to be unique.
-      - `kustomization.yaml`: Kustomization file including a single resource, which is a sibling file within the folder. Can pull in `components` for additional bootstrapping.
-      - `<RESOURCE_KIND_SINGULAR_NAME>.yaml`: **Single** resource manifest with a `.spec` that can be deployed to all the clusters.
-- `components`: Bootstrapping components pulled into `base` (this is different to the standard usage in Kustomize!).
-  - `<DESCRIPTIVE_COMPONENT_NAME>`: Folder describing intention of the component. Can be a grouping folder for additional nested components (by the subject), e.g. `project-admin-rolebinding` component is expected to define project admin `RoleBinding` and within this folder there are nested components for each specific user group.
-    - `kustomization.yaml`: Kustomization file including a all the resources required for this component. All resources are sibling files to this file.
-- `overlays`: Contains an overlay per managed cluster. Does not add any additional manifest. Only overlaying manifests from `base` is supported.
-  - `<MANAGED_CLUSTER_NAME>`: A folder per each managed cluster defining. Path to this folder is specified in the `Application` resource for cluster management application.
-    - `kustomization.yaml`: Cluster specific Kustomization file with a list of all resources pulled from base. This resource list is ordered alphabetically so it matches the order as it appears in file tree.
-    - Additional overlayed files, secret generators etc.
+This directory adheres to the following structure:
+
+```text
+- base                                                      Contains all the actual resource manifests which can be deployed to any clusters. Manifests are required to be stored in a format that can be applied to all the clusters (no cluster specific `.spec`s).
```

Comment:
Can you please maybe convert this into a tree style syntax? I think it may make it better readable.

Something like 

```
cluster-scope
├── base
│   ├── ...
│   └── <RESOURCE_KIND_API_GROUP>
│       └── <RESOURCE_KIND_PLURAL_NAME>
│           └── <RESOURCE_NAME>
│               ├── <RESOURCE_KIND_SINGULAR_NAME>.yaml
│               └── kustomization.yaml 
├── components
│   ├── ...
│   └── <DESCRIPTIVE_COMPONENT_NAME>
│       ├── ...
│       └── kustomization.yaml 
└── overlays
    ├── ...
    └── <MANAGED_CLUSTER_NAME> 
```

I admit that formatting this is tricky. Maybe we can convert the explanations/comments into a separate markdowns paragraphs below the tree structure like:

```markdown
# `base` folder

Contains all the actual...

## `<RESOURCE_KIND_API_GROUP> ` folder

The api group...

### `<RESOURCE_KIND_PLURAL_NAME>` folder

Each resource kind managed by this application...
```

WDYT? The current (or the previous version of this doc) is not very pleasing to read to me - I wouldn't feel like reading it.

Reply:
Yeah I agree, let's go with this suggestion

Reply:
Updated, let me know if this is better.

Reply:
Much better :+1: